recipes/sentry-breakpad/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.28.0"
+
+
+class SentryBreakpadConan(ConanFile):
+    name = "sentry-breakpad"
+    description = "A set of client and server components which implement a crash-reporting system."
```

Comment:
Only client in this case?

Reply:
Yes indeed.
I don't think it matters much, but I'll change it.